span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -234,7 +234,7 @@ private Event buildEvent(
       attributeFieldMap.put(
           RawSpanConstants.getValue(JaegerAttribute.JAEGER_ATTRIBUTE_SERVICE_NAME),
           AttributeValueCreator.create(jaegerSpan.getProcess().getServiceName()));
-      jaegerFieldsBuilder.setServiceName(jaegerSpan.getProcess().getServiceName());
+      eventBuilder.setServiceName(jaegerSpan.getProcess().getServiceName());
```

Comment:
quick test please?

Reply:
also do you want to still set the service name in the jaeger fields builder?

Reply:
@tim-mwangi Since we added first class field, another one on jaegerFields will be confusing so we removed that field.